src/facebook_example/bot.clj

```diff
@@ -28,6 +29,17 @@
       ; If no rules apply echo the user's message-text input
       :else (reaction/echo message-text))))
 
+(defn on-quick-reply [event]
+  (println "on-quickreply event:")
```

Comment:
What about moving the `println`s here (and in the other `on-…` functions) into `process-event`? I'm torn, on the one hand there's less clutter for the Lemmings on the other hand it is super nice to see "oh here is where the logging happens"